vizro-ai/mkdocs.yml

```diff
@@ -9,14 +9,16 @@ nav:
       - Run methods: pages/user-guides/run-vizro-ai.md
       - Create advanced charts: pages/user-guides/create-advanced-charts.md
       - Use different languages: pages/user-guides/use-different-languages.md
-  - Explanation:
+  - Find out more:
```

Comment:
Fine with this change, shall we make it consistent across both docs?

Reply:
Can we also be consistent in terms of icons, ie not use the same icon once for the "Explanation" section, and once for the API

Reply:
I can certainly make those changes.

Before I do though, I was wondering @maxschulz-CaOL whether we should just choose icons for the four different types of diataxis docs? But the problem with that is that _we_ know what it means but nobody looking at it without diataxis knowledge does. So maybe just commonly used icons that reflect the kind of information inside the section?